examples/alchemical_model/README.rst

```diff
@@ -0,0 +1,2 @@
+Basic usage of the Alchemical Model CLI
```

Comment:
Is this needed? @PicoCentauri

Reply:
If this is using sphinx-gallery, there needs to be a readme, but it can be empty